src/gei/Commands/MigrateRepoCommand.cs

```diff
@@ -103,6 +104,18 @@ public MigrateRepoCommand(OctoLogger log, ISourceGithubApiFactory sourceGithubAp
                 IsRequired = false,
                 Description = "Synchronously waits for the repo migration to finish."
             };
+            var githubSourcePat = new Option<string>("--github-source-pat")
+            {
+                IsRequired = false
+            };
+            var githubPat = new Option<string>("--github-pat")
```

Comment:
I'd suggest calling this --github-target-pat, the env var is GH_PAT because we wanted to be consistent across many commands with env var names. For in-line args that matters less, and would be more consistent with --github-source-org --github-target-org

Reply:
Addressed in e4c861708a7a68437149e65c058ee3825a020e0f